### DISCUSSION TITLE
Keep vertical tab bar collapsed on hover/click

### DISCUSSION THREAD

**Discussion by wealstarr** [Floorp-Projects/Floorp#1406] [Feature Request]
Hi,

Thank you for the amazing work. in Floorp version 11.17.8 I have chosen to collapse the tab bar. When I hover or click over the tabs, tab bar expands. Will it be possible to add an option to prevent it from expanding on hovering/clicking and keep it collapsed at all time ?

Kind regards

**Comment by tangowithfoxtrot** [comment]
I would also really like to see this as an option. This is my only frustration with Floorp, as it moves the tabs that I'm trying to click as I'm hovering, and makes the page content move around when the tab expands on-hover.
